src/main/java/domain/LottoNumberList.java

```diff
@@ -0,0 +1,34 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoNumberList {
+
+    private static final String SPLIT_CHAR = ",";
+    private List<Integer> numbers = new ArrayList<>();
```

Comment:
```suggestion
    private static final String SPLIT_CHAR = ",";
    
    private List<Integer> numbers = new ArrayList<>();
```

1. 클래스명에 자료구조 명은 지양해주세요
2. numbers 초기화에서 생성자를 쓴 것과 안 쓴 것과 어떻게 다를까요?